notifications_utils/postal_address.py

```diff
@@ -0,0 +1,122 @@
+from notifications_utils.countries import UK, Country, CountryNotFoundError
+from notifications_utils.countries.data import Postage
+from notifications_utils.formatters import (
+    normalise_lines,
+    remove_whitespace_before_punctuation,
+)
+from notifications_utils.recipients import (
+    is_a_real_uk_postcode,
+    first_column_headings,
+    format_postcode_for_printing,
+)
+
+
+address_lines_1_to_6_and_postcode = [
+    # The API only accepts snake_case placeholders
+    line.replace(' ', '_') for line in first_column_headings['letter']
+]
+address_lines_1_to_6 = address_lines_1_to_6_and_postcode[:-1]
+address_line_7 = 'address_line_7'
```

Comment:
It has to be hard coded at the moment because no other parts of the codebase know about `address_line_7` yet. For example the code that processes CSVs will still only look at:
```csv
address_line_1, address_line_2, address_line_3, address_line_4, address_line_5, address_line_6, postcode
```

Reply:
Hmm, ok, makes sense.

Just would that not mean that this hard-coded value will be chosen over postcode field value as per this test? https://github.com/alphagov/notifications-utils/pull/713/files#diff-e5050dea996611c8bdf32cfe6e8214cdR1792

Reply:
Ah no, it’s a hard coded key, not value.

Reply:
ahh, ok, makes more sense now, thanks. The variable names are a bit misleading then 🤔 

Also what would be the best way to test the code in practice?

Reply:
Renamed the variables in 74310cb

Reply:
👌